conf/local.conf.sample

```diff
@@ -38,20 +38,11 @@ MACHINE ??= "x64"
 
```

Comment:
I've never understood, can you explain the purpose of these .sample files? Are they just suggested base .conf files that anyone can start from?

Reply:
When you source the initialization file in the nilrt.git repo, it will create your build directory if it doesn't already exist. If that build directory doesn't contain a `:conf/bblayers.conf` or `:conf/local.conf` file, they will be copied from the directory you [define](https://github.com/ni/nilrt/blob/nilrt/master/sumo/ni-oe-init-build-env#L1) in your environment's `TEMPLATECONF` variable - which is meta-nilrt, in our case.

The `.sample` files are the template that is copied. So, in a sense, these are the *default* values for those local files.